SearchExampleSwift/ResultsViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  SearchResultsControllerViewController.swift
+//  Search
+//
+//  Created by David Haynes on 10/03/2016.
+//  Copyright © 2016 Ordnance Survey. All rights reserved.
+//
+
+import UIKit
+import Search
+import Fetch
+
+class ResultsViewController: UITableViewController {
+
+    var searchService: SearchService!
+    var results: [SearchResult]!
+
+    override func viewDidLoad() {
+        let apiKey = NSBundle.mainBundle().URLForResource("APIKEY", withExtension: nil).flatMap { url -> String? in
+            do {
+                return try String(contentsOfURL: url)
+            } catch {
+                return nil
+            }
+        } ?? ""
+
+        self.searchService = SearchService(apiKey: apiKey)
```

Comment:
Yeah good spot, had this in an `init` and got left when I switched back to `viewDidLoad`

Other use is inside a closure.